src/pages/snippets/deferred-loading-in-angular.mdx

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Same point here. Please push this off until the RFC is closed, and we actually have an initial implementation.

Reply:
@santoshyadavdev  Can you Suggest here,